pom.xml

```diff
@@ -4,15 +4,15 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.50</version>
+        <version>4.10</version>
         <relativePath />
     </parent>
     <artifactId>pagerduty</artifactId>
     <version>0.6.2-SNAPSHOT</version>
     <packaging>hpi</packaging>
     <properties>
         <java.level>8</java.level>
-        <jenkins.version>2.176.4</jenkins.version>
+        <jenkins.version>2.190.1</jenkins.version>
```

Comment:
Bumped parent pipeline and jenkins version to simplify testing via `mvn hpi:run`.  These are not explicitly required, as the change is likely backwards compatible.